tool/tctl/common/user_command.go

```diff
@@ -46,8 +46,8 @@ type UserCommand struct {
 	// DELETE IN (7.0)
 	// We keep legacy flags used in tctl users add --k8s-users command
 	legacyAllowedLogins string
-	legacyKubeUsers     string
-	legacyKubeGroups    string
+	kubeUsers           string
+	kubeGroups          string
```

Comment:
nit: maybe separate these from the `DELETE IN (7.0)` comment. Not sure what the comment is saying about `--k8s-users` either